benchmarks/markdown_id/package.json

```diff
@@ -51,6 +51,6 @@
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1",
     "bench": "rm -r markdown-pages; NUM_PAGES=${NUM_PAGES:-2000} node md.generate.js; gatsby clean; node --max_old_space_size=2000 node_modules/.bin/gatsby build",
     "benchnb": "gatsby clean; node --max_old_space_size=2000 node_modules/.bin/gatsby build",
-    "postinstall": "NUM_PAGES=${NUM_PAGES:-2000} node md.generate.js"
+    "postinstall": "rm -rf markdown-pages && gatsby clean && NUM_PAGES=${NUM_PAGES:-2000} node md.generate.js"
```

Comment:
Wouldn't this crash if the `markdown-pages` doesn't exist? I would use `;` after the `rm`

Reply:
No, that's actually what the `-f` is for.
From the man page:
```
-f, --force ignore nonexistent files and arguments, never prompt
```

Also if we ignored an error here it might mean that the benchmark was just wrong later (for example, if it actually ran with too many pages).

Reply:
Ah that's a good point :)